core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11970,5 +11970,28 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
 
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
+  
+  @Test public void testCountSetWithLiteralParameter() {
+    RelBuilder builder = relBuilder().scan("EMP");
```

Comment:
Please remove .scan("EMP"), as here we only use literal value. So there is no need to use a table scan.
Instead of this use builder.values(new String[]{""}, 1) below, while Relnode creation .

Reply:
Done